new/components/amqpinflux/scripts/create.sh

```diff
@@ -27,4 +44,5 @@ sudo useradd --shell /sbin/nologin --home-dir "${AMQPINFLUX_HOME}" --no-create-h
 ctx logger info "Fixing permissions..."
 sudo chown -R "${AMQPINFLUX_USER}:${AMQPINFLUX_GROUP}" "${AMQPINFLUX_HOME}"
 
-configure_systemd_service "amqpinflux"
\ No newline at end of file
+configure_systemd_service "amqpinflux"
+inject_service_env_var "(( amqp_cert_path ))" "${AMQP_CERT_PATH}" "amqpinflux"
```

Comment:
If would try and refrain from using inject_service_env_var as it's kinda hackish and set a runtime property and use it using download-resource-and-render. something like this:

```shell
# setting the property
ctx instance runtime_properties amqp_cert_path "${AMQP_CERT_PATH}"
```

and then inside the new/components/amqpinflux/config/cloudify-amqpinflux file write:

AMQP_CA_CERT_PATH="{{ ctx.instance.runtime_properties.amqp_cert_path }}"

This would be cleaner in the long run.